src/app/shared/code-editor-list/code-editor-list.component.scss

```diff
@@ -5,3 +5,7 @@
 .link-button {
   color: inherit;
 }
+
+.add-file-button {
+  left: 0.5rem;
```

Comment:
just use pl-2 if possible

Reply:
The button is not affected by pl-2, so instead I have used ml-2.